src/RequestFactory.php

```diff
@@ -150,7 +150,7 @@ private function buildBody(array $post, array &$headers, array $placeholders, ar
         }
 
         if ($this->isJson($headers)) {
-            return $this->replace(json_encode($post), $placeholders, $values);
+            return json_encode($this->replace($post, $placeholders, $values));
```

Comment:
You should update the docblock on \Cmp\Http\RequestFactory::replace

Reply:
Sure